code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -177,6 +178,7 @@
 	hitcost = 150
 	attack_verb = list("poked")
 	slot_flags = null
+	spawn_cell = FALSE
```

Comment:
I don't think that's a good way to solve it. Most objects just remove their cells in `Created` call or something like that.

Reply:
Better to not create the cell in the first place, than to create the cell then delete it

Reply:
I might have a idea to it, set the suitable_cell to null in  /obj/item/weapon/melee/baton/cattleprod and then set to /obj/item/weapon/cell/medium in New() ?

Reply:
No

Reply:
Well, this is all I could think to this problem. That's it.